### ISSUE TITLE
`+` operator to create sequence of actions

### ISSUE THREAD

**Issue by jcornaz** [libktx/ktx#172]
I think a `+` operator to easily create a sequence of actions would be nice

Example:
```kotlin
val sequence = moveTo(200f, 200f, 0.5f) + rotateTo(90f, 0.5f) + fadeOut(0.5f) + remove()
```

**Comment by czyzby** [comment]
It could be similar to `then` infix method, but it should not mutate sequence actions.

**Comment by jcornaz** [comment]
>It could be similar to `then` infix method, but it should not mutate sequence actions.

However I personally think that `+` and `then` should do exactly the same. (and not mutate the receiver)

Using `then` or `+` would only be a style choice (like using `crs` or `x` with vectors)

But a `+=` may mutate the receiver.

```
val sequence1 = moveTo(200f, 200f, 0.5f) + rotateTo(90f, 0.5f) + fadeOut(0.5f) 
val sequence2 = moveTo(200f, 200f, 0.5f) then rotateTo(90f, 0.5f) then fadeOut(0.5f) // should produce the same result as above

sequence2 += remove() // One could still mutate a sequence using +=
```

**Comment by jcornaz** [comment]
>Do you think it makes sense to add an operator for parallel actions? I'm afraid it would be less obvious than in case of sequences, but % or / could work.

No. I don't see the point of a non-obvious operator for parallel actions. In this case `parallelTo` is fairly enough.

**Comment by czyzby** [comment]
`|` would be more obvious, but it's not available in Kotlin `1.3`.
